apps/alpha-liquidator/README.md

```diff
@@ -62,6 +62,8 @@ Below are the environment variables used by the application, along with their ex
 
 - `MARGINFI_ACCOUNT_WHITELIST`: A comma-separated string of MarginFi account public keys to be whitelisted. It gets transformed into an array of `PublicKey` objects. This field is optional.
 
+- `JUP_SWAP_BASE_URL`: The Jupiter Swap v6 API endpoint URL as a string. Grab a free endpoint from [jupiterapi.com](https://www.jupiterapi.com) or a paid plan from [QuickNode](https://marketplace.quicknode.com/add-on/metis-jupiter-v6-swap-api).
```

Comment:
Please remove any references of 3rd party vendors.

Reply:
Remove the link to free endpoint or link to paid endpoint?

Reply:
Both

Reply:
No thanks. I think I'll just close the PR. Not in the spirit of open source to sensor. 😎